public/index.html

```diff
@@ -18,107 +22,265 @@
+                <li>Take one tablet daily</li>
+                <li>Prescribed by Dr. Ruth Parker</li>
+                <li>Pharmacy: Rite Aid #25335</li>
+                </ul>
+              </ol>
+            </li>
+            <li class="bubble__florence">Is that right?</li>
+            <li class="bubble__patient">Yep</li>
+            <li class="bubble__florence-tail">Ok, added</li>
+        </ol>
+        <img class="solution__section__right__avatar--2" src="assets/images/florence-avatar.svg">
+      </div>
+    </section>
+
+    <section role="region" class="solution__section--3">
+      <div class="solution__section__left">
+        <img class="solution__section__img3-bg" src="assets/images/section3-illo-bg.jpg">
+        <img class="solution__section__img3" src="assets/images/section3-illo.svg">
+        <h3 class="solution__section__hdr">She checks up on you.</h3>
+        <p class="solution__section__content">As you start taking new medications, Florence will follow up to see if you’re experiencing side effects.</p>
+        <p class="solution__section__content">She can offer additional information to help you understand if your experience is typical—or if you should talk with your doctor..</p>
+      </div>
+      <div class="solution__section__right">
+        <ol class="bubble">
+          <li class="bubble__florence">It’s been three days since you started taking Lisinopril. How’re you feeling?</li>
+          <li class="bubble__patient">a little weak and lightheaded</li>
+          <li class="bubble__florence">Feeling lightheaded is a common side effect of Lisinopril. But you should let Dr. Ruth Parker know as soon as possible</li>
+          <li class="bubble__florence-tail">Here’s her number: <a class="bubble__number" href="">(919) 545-2020</a></li>
+        </ol>
+        <img class="solution__section__right__avatar--3" src="assets/images/florence-avatar.svg">
+      </div>
+    </section>
+
+    <section role="region" class="solution__section--4">
+      <div class="solution__section__left">
+        <img class="solution__section__img4-bg" src="assets/images/section4-illo-bg.jpg">
+        <img class="solution__section__img4" src="assets/images/section4-illo.svg">
+        <h3 class="solution__section__hdr">She helps you remember.</h3>
+        <p class="solution__section__content">With complex regimens, days often blur together. Florence will show you what you’ve taken today — and what’s left to take. </p>
+      </div>
+      <div class="solution__section__right">
+        <ol class="bubble">
+          <li class="bubble__patient">did I take my meds today?</li>
+          <li class="bubble__florence--tail">
+            Here's what I have for you today:
+            <ol>
+              <li><img class="emoji" src="assets/images/check.svg">Lisinopril</li>
+              <li><img class="emoji" src="assets/images/check.svg">Coumadin</li>
+              <li><img class="emoji" src="assets/images/check.svg">Zoloft</li>
```

Comment:
you'll want some good `alt` text on these, since the emojis do add some context/content.